Volume/Supporting/AboutConstants.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  AboutConstants.swift
+//  Volume
+//
+//  Created by Vin Bui on 4/25/21.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+
+struct AboutConstants {
+    
+    static let missionStatement = "Created by Cornell AppDev, Volume aims to amplify the voices of student publications and organizations by making it easy and accessible to digest student created content.\n\nVolume believes in the unique power that lies in the sharing and distribution of content by students and for students to spark conversation, experiences, and connections that improve the collegiate experience."
+    
+    static let volumeRoster: [PodRoster] = [
```

Comment:
I personally am against having rosters in app since they are difficult to maintain